lib/src/models/post.dart

```diff
@@ -76,6 +76,7 @@ class PostModel with _$PostModel {
     required bool? canAuthUserModerate,
     required NotificationControlStatus? notificationControlStatus,
     required List<String>? bookmarks,
+    required bool? read,
```

Comment:
Would it make sense to make this non-nullable, so you don't have to check `read != null` multiple times? Mbin and PieFed already use a non-null default (`false`), and Lemmy could fallback to `false` if `null` is provided. I don't know for sure, but it's possible Lemmy's `read` field is only `null` when you are a guest.